.github/workflows/nodejs.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         os: [macOS-latest, ubuntu-latest, windows-latest]
         node-version: [10.x, 12.x]
-        python-version: [3.7.3]
+        python-version: [3.7.4]
```

Comment:
Strangely, `setup-python` does not support python `3.7.3` so we use this version here.